frontend/src/metabase/services.js

```diff
@@ -314,10 +313,6 @@ export const MetabaseApi = {
   // table_fields:                GET("/api/table/:tableId/fields"),
   table_fks: GET("/api/table/:tableId/fks"),
   // table_reorder_fields:       POST("/api/table/:tableId/reorder"),
-  table_query_metadata: GET(
```

Comment:
I believe we use RTK query instead
https://github.com/metabase/metabase/blob/25889574222cc809cb345da89ca35bc8f117e78d/frontend/src/metabase/api/table.ts#L38-L45